src/reducers/userProfileReducerDB.js

```diff
@@ -7,8 +7,9 @@ const userProfileReducerDB = (currentUser = initialUser, action) => {
       return action.user;
     case "ADD_USER":
       return action.newUser;
-    case "FETCHED_CURRENT_PROFILE":
+    case "FETCHED_CURRENT_PROFILE": {
```

Comment:
do you need these curly braces? switch statement cases shouldnt i think. at least from my knowledge

Reply:
I guess it's kind of a habit of mine . I don't think the curly braces are necessary.